ui/src/components/RoleForm/index.jsx

```diff
@@ -108,8 +110,22 @@ export default class RoleForm extends Component {
     this.props.onRoleSave(role, roleId);
   };
 
+  handleDialogActionError = error => {
+    this.props({ dialogError: error });
```

Comment:
`this.props` is not a function. It should probably be `this.setState`. Also, make sure you to test the functionality to see if you are able to see and error in the dialog when trying to delete a role.

Reply:
@helfi92 this function is not used anywhere so I have removed it for now.